Causes the Pre to be recognized as a mass storage media player when connected to a linux desktop, without spoofing an ipod device.
Compatible with Banshee, Ryhthmbox, and several other open-source media players.

Works in both USB mode, and Media Sync mode.

Index: /media/internal/.is_audio_player
===================================================================
--- /dev/null
+++ /media/internal/.is_audio_player
@@ -0,0 +1,3 @@
+name=PalmPre
+audio_folders=Music/, AmazonMP3/, Videos/
+output_formats=audio/mp3,audio/aac,video/mp4,video/quicktime,video/mp4-generic,video/mpeg4,application/sdp
